src/theme.cpp

```diff
@@ -431,7 +431,7 @@ theme::label::label()
 
 theme::label::label(std::size_t sw, std::size_t sh, const config& cfg)
 	: object(sw, sh, cfg)
-	, text_(cfg["prefix"].str() + cfg["text"].str() + cfg["postfix"].str())
+	, text_(cfg["prefix"].str() + cfg["prefix_literal"].str() + cfg["text"].str() + cfg["postfix_literal"].str() + cfg["postfix"].str())
```

Comment:
What is prefix_literal and postfix_literal?

Reply:
It's pretty much exactly what it says – a literal, untranslated string that prefixes or postfixes the main string.

Reply:
Why is that being added with this PR?

Reply:
It was so that certain things in the theme didn't have to be translated. In particular I think there were several cases of strings consisting of only spaces.